telemetry/specs/telemetry_report.mdk

```diff
@@ -398,6 +399,45 @@ particular deployment.
   : The device local time when the packet was first received on the ingress
     physical or logical port, in nanoseconds.
 
+## Telemetry Extensions Header
```

Comment:
There are multiple 'extension' hdrs, e.g., the domain-specific extension in INT spec.
How about calling this one as "Report Extensions" header?
'Report' seems the main keyword that distinguishes this extension from other extensions.